devenv/docker/blocks/sensugo/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+  sensu-backend:
+    image: sensu/sensu:latest
+    container_name: sensu-backend
+    ports:
+      - "3000:3000"
```

Comment:
Can the binding be changed here? 3000 is the default dev port for Grafana, so will conflict with a locally running dev instance.

Reply:
Yes, port 3000 is the Web UI.  Which I doubt you would need/use.  I will change to 3080.